internal/jobworker/cgroup/cgroup_test.go

```diff
@@ -0,0 +1,233 @@
+package cgroup
+
+import (
+	"bufio"
+	"errors"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"testing"
+)
+
+func TestCleanup(t *testing.T) {
+	if !isRoot() {
+		t.Skip("must be root to run")
+	}
+
+	dir := t.TempDir()
+	service, err := NewService(WithMountPath(dir))
+	if err != nil {
+		t.Errorf("%+v", err)
+		return
+	}
+
+	if _, err := os.Stat(service.path); err != nil {
+		t.Error(err)
+	}
+
+	expected := []string{
+		cpu,
+		io,
+		memory,
+	}
+	controllers, err := readControllers(service.path)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if !reflect.DeepEqual(controllers, expected) {
```

Comment:
If we are testing the cleanup (TestCleanup), then why the assertions along the way? Others tests should take care of those.

Reply:
Updated test name to better reflect test. Test ensures root and jobworker base cgroup are properly configured, hence the controllers check.